tests/fixtures/test_fixtures.py

```diff
@@ -39,7 +39,7 @@ def test_load_pages(app):
         force=True,
     ).load()
 
-    pages = Page.query.all()
+    pages = current_pages_service.search(system_identity)._results
```

Comment:
Minor: Single leading underscores are generally used to defined "internal" attributes, meaning that we should avoid accessing them outside of the class.